Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:stretch
```

Comment:
No sure this program needs Debian
You should use Alpine

Docker content trust works fine, you can read how to automate or use it here: https://github.com/sudo-bot/action-docker-sign